controllers/fluentbit_controller.go

```diff
@@ -182,6 +183,49 @@ func (r *FluentBitReconciler) mutate(obj client.Object, fb fluentbitv1alpha2.Flu
 			}
 			return nil
 		}
+	case *rbacv1.Role:
+		expected, _, _ := operator.MakeScopedRBACObjects(fb.Name, fb.Namespace)
+
+		return func() error {
+			o.Name = expected.Name
```

Comment:
@wanjunlei what do you think of this PR?
Do we need to add below code to all cases?
```go
			if err := ctrl.SetControllerReference(&fb, o, r.Scheme); err != nil {
				return err
			}
```